Makefile

```diff
@@ -48,7 +48,7 @@ cw_alarm_agent_clean:
 cw_alarm_agent_distclean: CW_ALARM_AGENT_clean
 
 cw_mib: env
-	${ENV_DIR}/bin/python ${ROOT}/mib-generator/cw_mib_generator.py && mv ${ROOT}/mib-generator/PROJECT-CLEARWATER-MIB ${ROOT}/clearwater-snmp-alarm-agent.root/usr/share/clearwater/mibs/
+	${ENV_DIR}/bin/python ${ROOT}/mib-generator/cw_mib_generator.py ${ROOT}/clearwater-snmp-alarm-agent.root/usr/share/clearwater/mibs/
```

Comment:
Sorry, I'm not quite clear - are you suggesting that we shouldn't make this change?

I'm happy to do so, but I don't think it will help much - a quick browse of our other Makefiles shows variables without quote marks being used in lots of them. I suspect a file structure containing spaces will break most of our builds.